.travis.yml

```diff
@@ -11,6 +11,7 @@ install:
 - pip install -r requirements.txt
 - pip install coveralls readme_renderer
 script:
+- python setup.py install
```

Comment:
Well the transport can be mocked int the same way as in https://github.com/ethereum/pyethapp/pull/168 so this will not be necessary.